examples/cosmos/main.go

```diff
@@ -73,7 +73,7 @@ func requireNil(err error) {
 func main() {
 
 	c := cosmostest.SetupCollection(log.New(os.Stderr, "", log.LstdFlags), cosmostest.Config{}, "mycollection", "userId")
-	defer cosmostest.Teardown(c)
+	defer cosmostest.TeardownCollection(c, true)
```

Comment:
I'd prefer if we introduce this argument when we need it, and just hardcode to true now